packages/angular/projects/ui-angular/src/lib/common/types/component-types.ts

```diff
@@ -4,7 +4,7 @@ import { AuthFormData } from '@aws-amplify/ui';
 /**
  * Maps custom components from customer to the name of the component it's overriding.
  */
-export type CustomComponents = Record<string, TemplateRef<any>>;
+export type CustomComponents = Record<string, TemplateRef<unknown>>;
```

Comment:
For context, the generic type in `TemplateRef` corresponds to the type of any [template contexts](https://angular.io/api/common/NgTemplateOutlet#description). Template are data that you can pass from template outlet (ie. "renderer") to the template.

Since we do not know what the type of context between all possible slots, we previously used `any`. Changing this to `unknown`.

Reply:
This is not customer facing, because `ngTemplateOutlet` is actually [not typed anyway](https://github.com/angular/angular/blob/12.2.17/packages/common/src/directives/ng_template_outlet.ts#L50).